core-bundle/src/Security/Authentication/ContaoLoginAuthenticationListener.php

```diff
@@ -63,7 +63,7 @@ protected function attemptAuthentication(Request $request): ?TokenInterface
         $username = $request->request->get('username');
         $password = $request->request->get('password');
 
-        if (!\is_string($username) && (!\is_object($username) || !method_exists($username, '__toString'))) {
```

Comment:
Are you sure this can be changed since we still support Symfony 4.4?

Reply:
Yes.